app/helpers/job_applications_helper.rb

```diff
@@ -40,6 +40,15 @@ def job_application_qualified_teacher_status_info(job_application)
     end
   end
 
+  def job_application_jobseeker_profile_info(job_application)
+    profile = job_application&.jobseeker&.jobseeker_profile
+    if profile&.has_teacher_reference_number.present? && profile&.has_teacher_reference_number == "yes"
```

Comment:
The second condition also returns false if the `profile&.has_teacher_reference_number` is not present.
```suggestion
    if profile&.has_teacher_reference_number == "yes"
```